spec/models/location_spec.rb

```diff
@@ -743,7 +743,7 @@
     it "does add an error when the number of units is invalid" do
       location.units = nil
       location.valid?(:units)
-      expect(location.errors.count).to eq(1)
+      expect(location.errors.count).to eq(2)
```

Comment:
Could we also add a tests specifically for the number range?

Reply:
Yup added, a few to test invalid and valid values around the boundary (greater than or equal to 1) and others.